nodes/sample_host.json

```diff
@@ -40,9 +40,10 @@
       "redirect_domain_names": ["<domain name>", "<domain name>", "<...>"],
       "ruby_version": "2.1.0",
       "ssl_info": {
-        "key": "<ssl key>",
-        "crt": "<ssl crt>"
-        },
+        "enabled": true,
+        "certificate": "The cert file, optional",
```

Comment:
agree, but this is not the full path to cert files but a cert file name only. These files should be copied to `vendor/cookbook/rails/files/default/certificates` before uploading. For example:
```
"ssl_info": {
  "enabled": true,
  "certificate": "my_cert.crt", # if not given, the app's name will be used, for ex: intercity_sample_app.crt"
  "certificate_key": "my_cert.key"
}
}
```